build.sbt

```diff
@@ -192,3 +161,9 @@ lazy val benchmark = (project in file("benchmark"))
     test in assembly := {},
     assemblyOutputPath in assembly := file("./utils/bin/firrtl-benchmark.jar")
   )
+
+// Aliases
```

Comment:
Why is this part of a PR for upgrading to `2.13`?
Could you please remove this to make this PR easier to review?